src/utils/complexity.js

```diff
@@ -1,8 +1,16 @@
 import { log } from "dbc-node-logger";
-import queryComplexity from "graphql-query-complexity";
+import { parse, GraphQLError } from "graphql";
+
+import queryComplexity, {
+  getComplexity,
+  simpleEstimator,
+  directiveEstimator,
+} from "graphql-query-complexity";
 
 import config from "../config";
 
+// THIS WILL BE DEPRECATED IN FUTURE
```

Comment:
when is the future now ! 

enten en mere uddybende kommentar - altså sådan noget med når vi skal til at implementere SLA og  det ny kompleksitetsberegning er valideret -- så er the the future

eller en story på backlog - a'la  SLA implementation - husk at slette den gamle  kompleksitetsberegning (markeret i koden)